templates/repo/sub_menu.tmpl

```diff
@@ -2,7 +2,7 @@
 	<div class="ui two horizontal center link list">
 		{{if and (.Repository.UnitEnabled $.UnitTypeCode) (not .IsBareRepo)}}
 			<div class="item{{if .PageIsCommits}} active{{end}}">
-				<a href="{{.RepoLink}}/commits/{{EscapePound .BranchName}}"><i class="octicon octicon-history"></i> <b>{{.CommitsCount}}</b> {{.i18n.Tr "repo.commits"}}</a>
+				<a href="{{.RepoLink}}/commits/branch/{{EscapePound .BranchName}}"><i class="octicon octicon-history"></i> <b>{{.CommitsCount}}</b> {{.i18n.Tr "repo.commits"}}</a>
```

Comment:
Shouldn't there needs to be url depending on what page you are browsing (tags/commits/branches)?

Reply:
You are right, I have to do check for `tags`.